ibm/resource_ibm_kms_key.go

```diff
@@ -94,6 +94,13 @@ func resourceIBMKmskey() *schema.Resource {
 				Computed:    true,
 				Description: "Crn of the key",
 			},
+			"expiration_date": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The date the key material expires. The date format follows RFC 3339. You can set an expiration date on any key on its creation. A key moves into the Deactivated state within one hour past its expiration date, if one is assigned. If you create a key without specifying an expiration date, the key does not expire",
+				ForceNew:    true,
+				Default:     "Default",
```

Comment:
Don't set any Default value....since the API doesn't  have any Default value

Reply:
Ok. Will do so.